test/unit/helpers/xhr/FACTOR_ENROLL_push_success.js

```diff
@@ -0,0 +1,83 @@
+define({
```

Comment:
file name for this one is `FACTOR_ENROLL_push_success`, but the contents seem to be for `FACTOR_ENROLL_ACTIVATE_push`, which you have a file for this already. Maybe the name is wrong? Or can we reuse the other file above?

Reply:
I'll try and re-use `FACTOR_ENROLL_ACTIVATE_push .js`

Reply:
renamed the file